src/modules/wallet/reducer.ts

```diff
@@ -98,6 +107,25 @@ export function walletReducer(
         data: null
       }
     }
+    case FETCH_WALLET_REQUEST: {
+      return {
+        ...state,
+        loading: loadingReducer(state.loading, action)
+      }
+    }
+    case FETCH_WALLET_SUCCESS: {
+      return {
+        ...state,
+        loading: loadingReducer(state.loading, action),
+        data: action.payload.wallet
+      }
+    }
+    case FETCH_WALLET_FAILURE: {
+      return {
+        ...state,
+        loading: loadingReducer(state.loading, action)
```

Comment:
Should we need to return the `error` here? If so, Can we move this with `ENABLE_WALLET_FAILURE`?

Reply:
Nope, explained above.